common/unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -801,6 +804,49 @@ public static UTF8String concatWs(UTF8String separator, UTF8String... inputs) {
     return res;
   }
 
+  /**
+   * Return a locale of the given language and country, or a default locale when failures occur.
+   */
+  private Locale getLocale(UTF8String language, UTF8String country) {
+    try {
+      return new Locale(language.toString(), country.toString());
+    } finally {
+      return Locale.getDefault();
+    }
+  }
+
+  /**
+   * Splits a string into arrays of sentences, where each sentence is an array of words.
+   */
+  public ArrayList<ArrayList<UTF8String>> sentences(UTF8String language, UTF8String country) {
```

Comment:
This method is implemented by String not UTF8String, I think we should put it into another util object(maybe in scala) and takes String as arguments.